packages/web/src/pages/artist-coins-launchpad-page/components/ArtistCoinsTable.tsx

```diff
@@ -161,11 +161,9 @@ const renderBuyCell = (
   return (
     <Flex pr='xl' justifyContent='flex-end'>
       <Button
-        variant='tertiary'
+        variant='secondary'
         size='small'
-        css={{
-          boxShadow: '0 0 0 1px inset var(--harmony-border-default) !important'
-        }}
+        hoverColor='coinGradient'
```

Comment:
I had included the boxShadow css to get rid of the shadows. does this make the shadows come back? maybe screenshot?